packages/core/src/lib/handler/sms.handler.spec.ts

```diff
@@ -28,8 +29,36 @@ test('send sms should call the provider method correctly', async () => {
 
   expect(spy).toHaveBeenCalled();
   expect(spy).toHaveBeenCalledWith({
-    'content': 'Name: test name',
-    'to': '+1333322214'
+    content: 'Name: test name',
+    to: '+1333322214',
   });
   spy.mockRestore();
+
+  const spyTemplateFunction = jest
+    .fn()
+    .mockImplementation(() => Promise.resolve('test'));
+
+  emailHandler = new SmsHandler(
```

Comment:
I believe it should be smsHandler :) I now see that the mistake is also in the code above.